### PR TITLE
Add Laser Sniper, Add said Sniper to Syndi Vendor

### PR DESCRIPTION
**Pull request by Harrivchaw**
# What this PR does
See Title

# Why it should be added to the game
Laser gun variety

Sprites taken from /TG/

image
https://user-images.githubusercontent.com/74265710/129468952-3e851c57-edf4-47bb-b0c8-2a4405291c78.mp4

### CODE REVIEW

File: code/_core/datum/damagetype/ranged/bullet/sniper_50.dm [hunk 1]
```diff
@@ -13,7 +13,7 @@
 		BLUNT = 90
 	)
 
-	falloff = VIEW_RANGE + ZOOM_RANGE*5
+	falloff = VIEW_RANGE + ZOOM_RANGE*2
```
Comment (BurgerLUA) [inline code/_core/datum/damagetype/ranged/bullet/sniper_50.dm:16]: why was the falloff nerfed here?

File: code/_core/obj/item/clothing/head/hats/witchunter.dm [hunk 2]
```diff
@@ -1,16 +1,16 @@
-/obj/item/clothing/head/hat/witchunter
```
Comment (BurgerLUA) [inline code/_core/obj/item/clothing/head/hats/witchunter.dm:1]: I don't know why the witchhunter file was changed here. You might be experiencing an issue with newlines. See https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings on how to fix it.

**Review by BurgerLUA — CHANGES_REQUESTED** [review]
various changes are in this that shouldn't be in this.

File: code/_core/obj/item/clothing/head/hats/witchunter.dm (hunk 2 above)
Comment (Harrivchaw) [reply code/_core/obj/item/clothing/head/hats/witchunter.dm:1]: i have zero idea how to fix this

File: code/_core/datum/damagetype/ranged/bullet/sniper_50.dm (hunk 1 above)
Comment (Harrivchaw) [reply code/_core/datum/damagetype/ranged/bullet/sniper_50.dm:16]: wanted to differentiate between las sniper and heavy sniper, las sniper has the old faloff value, change them to be equal?

**Review by BurgerLUA — APPROVED** [review]

**Pull request merged**